example/test_solrcloud.yaml

```diff
@@ -14,13 +14,14 @@ spec:
   solrImage:
     tag: 8.2.0
   solrJavaMem: "-Xms1g -Xmx3g"
-  solrPodPolicy:
-    resources:
-      limits:
-        memory: "1G"
-      requests:
-        cpu: "65m"
-        memory: "156Mi"
+  customSolrKubeOptions:
```

Comment:
Thanks for making this fix, I had a branch that made these changes but I forgot to make a PR.